test/ngAria/ariaSpec.js

```diff
@@ -85,6 +85,28 @@ describe('$aria', function() {
       expect(element.attr('aria-checked')).toBe('false');
     });
 
+    it('should handle checkbox with string model values using ng(True|False)Value', function() {
+      var element = $compile('<input type="checkbox" ng-model="val" ng-true-value="\'yes\'">' +
+        'ng-false-value="\'no\'"'
```

Comment:
Yes, definitely! although it doesn't make a difference, since we don't test if the correct ng-false-value is set.